### PR TITLE
maintain separate requirements based on the environment

### PR DESCRIPTION
**Pull request by harshad16**
maintain separate requirements based on the environment
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Related Issues and Dependencies

Related-to: #134

### CODE REVIEW

File: f31-py37/Pipfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+name = "pypi"
+
+[packages]
+thamos = "*"
+micropipenv = {extras = ["toml"], version = "*"}
+
+[dev-packages]
+
+[requires]
+python_version = "3.7"
```
Comment (harshad16) [inline f31-py37/Pipfile:13]: kept the Pipfile and Pipfile.lock for easy maintenance. 
can be removed if feels like an unnecessary addition.

Comment (fridex) [reply f31-py37/Pipfile:13]: I think it can be removed. You can directly pass `Pipfile.lock` to micropipenv, without ``-deploy`` it will not need Pipfile.

Comment (harshad16) [reply f31-py37/Pipfile:13]: i will remove the pipfile and pipfile.lock
and let stay with the requirements.txt, as it is easy to use in other build tools.

**Review by fridex — COMMENTED** [review]
lgtm :+1:

File: f31-py37/Pipfile (hunk 1 above)
Comment (harshad16) [reply f31-py37/Pipfile]: i have made the changes and only kept the requirements.txt.

**Review by fridex — APPROVED** [review]
Awesome, thanks! :+1:

**Pull request merged**